rpm/nemo-qml-plugin-calendar-qt5.spec

```diff
@@ -47,3 +60,10 @@ rm -rf %{buildroot}
 %files tests
 %defattr(-,root,root,-)
 /opt/tests/nemo-qml-plugins-qt5/calendar/*
+
+%files lightweight
+%defattr(-,root,root,-)
+%attr(2755, root, privileged) %{_bindir}/calendardataservice
```

Comment:
Doesn't feel proper to have this kind of access right policy in nemo mw.

Then again not sure if this service belongs here to begin with as it's more than just a qml plugin.